test/drop_users.sql

```diff
@@ -3,6 +3,9 @@
 -- Note that dropping a non-existing user produces an error that aborts the
 -- script, so we first grant a harmless privilege to each user to ensure it
 -- exists.
+
+USE mysql;
```

Comment:
Err, this is for switching databases, right? Why switch database when you're editing users?